packages/grid/_modules_/grid/models/api/gridFilterApi.ts

```diff
@@ -0,0 +1,51 @@
+import { GridRowId, GridRowModel } from '../gridRows';
+import { GridFilterModel } from '../../hooks/features/filter/gridFilterModelState';
+import { GridFilterItem, GridLinkOperator } from '../gridFilterItem';
+
+/**
+ * The filter API interface that is available in the grid [[apiRef]].
+ */
+export interface GridFilterApi {
+  /**
+   * Shows the filter panel. If `targetColumnField` is given, a filter for this field is also added.
```

Comment:
**Side**

Great initiative with the plural, most of the other descriptions are singular, which is wrong.